src/Kernel32.Desktop/Kernel32+LANGID.cs

```diff
@@ -0,0 +1,448 @@
+﻿// Copyright (c) to owners found in https://github.com/AArnott/pinvoke/blob/master/COPYRIGHT.md. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+
+namespace PInvoke
+{
+    using System.Runtime.InteropServices;
+
+    /// <content>
+    /// Contains the language ID (LANGID) constants.
+    /// </content>
+    public partial class Kernel32
+    {
+        /// <summary>
+        ///     Creates a language identifier from a primary language identifier and a sublanguage identifier.
+        /// </summary>
+        /// <param name="usPrimaryLanguage">
+        ///     Primary language identifier. This identifier can be a predefined value or a value for a user-defined primary
+        ///     language. For a user-defined language, the identifier is a value in the range 0x0200 to 0x03FF. All other
+        ///     values are reserved for operating system use. For more information, see Language Identifier Constants and
+        ///     Strings.
+        /// </param>
+        /// <param name="usSubLanguage">
+        ///     Sublanguage identifier. This parameter can be a predefined sublanguage identifier or a user-defined
+        ///     sublanguage. For a user-defined sublanguage, the identifier is a value in the range 0x20 to 0x3F. All other
+        ///     values are reserved for operating system use. For more information, see Language Identifier Constants and
+        ///     Strings.
+        /// </param>
+        /// <returns>Returns the language identifier.</returns>
+        public static LANGID MAKELANGID(ushort usPrimaryLanguage, ushort usSubLanguage)
+        {
+            return new LANGID
+            {
+                PrimaryLanguage = (LANGID.PrimaryLanguageType)usPrimaryLanguage,
+                SubLanguage = (LANGID.SubLanguageType)usSubLanguage
+            };
+        }
+
+        /// <summary>
+        /// Represents a language identifier.
+        /// </summary>
+        [StructLayout(LayoutKind.Sequential, Pack = 1)]
+        public struct LANGID
+        {
+            internal const ushort PrimaryLanguageMask = 0x1FF;
+            internal const ushort SubLanguageMask = 0xFE00;
+            internal const int SubLanguageShift = 9;
+
+            internal ushort data;
+
+            public LANGID(ushort usLanguages)
+            {
+                this.data = usLanguages;
+            }
+
+            public enum PrimaryLanguageType : ushort
```

Comment:
Done, although I renamed the properties to avoid name duplication between them and the types.